lib/watir/elements/radio.rb

```diff
@@ -1,8 +1,12 @@
 module Watir
   class Radio < Input
-    def initialize(query_scope, selector)
-      super
+    def build
       @selector[:label] = @selector.delete(:text) if @selector.key?(:text)
+      super
+    end
+
+    def driver_scope
```

Comment:
What is different about `Radio` that requires this?

Reply:
Getting `RadioSet` to work was a pain. All that code made sense when I created it, but it was very confusing digging back into it. :-D I'm guessing I needed to have it here to get that to work, but I don't recall. I'll try removing it and see what happens. I should add more comments for these things in general.